process/block/serialize_test.go

```diff
@@ -55,3 +56,29 @@ func TestBlockProcessor_SerializeEpochInfoData(t *testing.T) {
 	require.Nil(t, err)
 	require.Equal(t, `{"accumulatedFees":"1","developerFees":"2"}`, buff.String())
 }
+
+func TestBlockProcessor_SerializeBlockEpochStartMeta(t *testing.T) {
+	t.Parallel()
+
+	bp, _ := NewBlockProcessor(&mock.HasherMock{}, &mock.MarshalizerMock{})
+
+	buff, err := bp.SerializeBlock(&data.Block{
+		Hash:            "11cb2a3a28522a11ae646a93aa4d50f87194cead7d6edeb333d502349407b61d",
+		Size:            345,
+		ShardID:         core.MetachainShardId,
+		EpochStartBlock: true,
+		SearchOrder:     0x3f2,
+		EpochStartInfo: &data.EpochStartInfo{
+			TotalSupply:                      "100",
+			TotalToDistribute:                "55",
+			TotalNewlyMinted:                 "20",
+			RewardsPerBlock:                  "15",
+			RewardsForProtocolSustainability: "2",
+			NodePrice:                        "10",
+			PrevEpochStartRound:              222,
+			PrevEpochStartHash:               "7072657645706f6368",
+		},
+	})
+	require.Nil(t, err)
+	require.Equal(t, `{"nonce":0,"round":0,"epoch":0,"miniBlocksHashes":null,"notarizedBlocksHashes":null,"proposer":0,"validators":null,"pubKeyBitmap":"","size":345,"sizeTxs":0,"timestamp":0,"stateRootHash":"","prevHash":"","shardId":4294967295,"txCount":0,"accumulatedFees":"","developerFees":"","epochStartBlock":true,"searchOrder":1010,"epochStartInfo":{"totalSupply":"100","totalToDistribute":"55","totalNewlyMinted":"20","rewardsPerBlock":"15","rewardsForProtocolSustainability":"2","nodePrice":"10","prevEpochStartRound":222,"prevEpochStartHash":"7072657645706f6368"}}`, buff.String())
```

Comment:
can you please add values in developer fees and other fields?

Reply:
done.